src/utils/canonicalizeTitle.js

```diff
@@ -0,0 +1,4 @@
+
+export const canonicalizeTitle = title => {
```

Comment:
This is basically the same thing as `utils/normalizeTitle.js`[1] I would suggest keeping only one and fixing it.

[1] `normalizeTitle` is not implemented well. It will only replace the first space with an underscore. It should do something like `title.replace(/\s+/g, '_')`

Reply:
so there are two things

normalizeTitle : convert all _ to ' '
canonicalizeTitle : convert all ' ' to _

Reply:
You're right